index.test.js

```diff
@@ -54,7 +59,7 @@ describe("eslint-ratchet", () => {
 
     setupMocks();
     expect(() => formatter(newResults, null, logger)).to.throw();
-    expect(JSON.stringify(messages)).to.equal(JSON.stringify(expectedMessages));
+    expectedMessages.forEach((message) => expect(messages).to.contain(message));
```

Comment:
I dont fully follow the change to this test. Iterating the messages and swapping the matcher result in being able to drop stringification?

Reply:
Yep good call out, it's not super clear.

Pre-changes this assertion was expecting that the messages logged match the expected cases **perfectly**, which had a couple of issues:
- formatting like you called out can be a bit off due to stringification
  - this isn't always the case however when the bug was fixed where the logger wasn't being used it became more obvious
- additional things may be tossed into the log that we don't care about
  - this has more to do with eslint and repo based config in general but I wanted to have the flexibility to replace those pieces without the tests here caring - they are about logging unique to the ratcheter after all 🤷‍♂️

Post-changes as long as the messages we expect are there we're good.